packages/KIcon/icons/icn-connections.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="24px" viewBox="0 0 18 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Combined Shape</title>
```

Comment:
omit the title so user's don't see this text when hovering over the icon. KIcon uses the icon name e.g. 'connections' when rendering.